README.md

```diff
@@ -168,19 +169,23 @@ Additionally, each of the main interface methods accept an optional options obje
 
 However, it is possible to *reopen* a database after it has been closed with <a href="#close"><code>close()</code></a>, although this is not generally advised.
 
+If no callback is passed, a promise is returned.
```

Comment:
Maybe: "native promise". Or mention that in the Promise Support section.

Reply:
addressed